.github/workflows/build-release.yml

```diff
@@ -68,95 +68,21 @@ jobs:
-          Expand-Archive -Force "zip\Particular.ServiceControl.Monitoring-${{ needs.release.outputs.majorminorpatch }}.zip" -DestinationPath "unzip\Particular.ServiceControl.Monitoring"
-
-          $transport = "${{ matrix.transport }}"
-          $srcTransport = "${{ matrix.transport-source}}"
-
-          if ( $srcTransport -eq $null -Or $srcTransport -eq "" ) {
-            $srcTransport = $transport
-          }
-      
-          New-Item -ItemType Directory -Force -Path ServiceControl.Transports.$transport/bin/Release/net472
-          Copy-Item -Path "unzip\Particular.ServiceControl\Transports\$srcTransport\*" -Destination "ServiceControl.Transports.$transport\bin\Release\net472" -Recurse
-
-          New-Item -ItemType Directory -Force -Path ServiceControl/bin/Release/net472
-          Copy-Item -Path "unzip\Particular.ServiceControl\ServiceControl\*" -Destination "ServiceControl\bin\Release\net472" -Recurse
-
-          New-Item -ItemType Directory -Force -Path ServiceControl.Persistence.RavenDb/bin/Release/net472
-          # No need to copy anything into this folder as it has already been handled during packaging
-
-          New-Item -ItemType Directory -Force -Path ServiceControl.Audit/bin/Release/net472
-          Copy-Item -Path "unzip\Particular.ServiceControl.Audit\ServiceControl.Audit\*" -Destination "ServiceControl.Audit\bin\Release\net472" -Recurse
-
-          New-Item -ItemType Directory -Force -Path ServiceControl.Audit.Persistence.RavenDb/bin/Release/net472
-          Copy-Item -Path "unzip\Particular.ServiceControl.Audit\Persisters\RavenDB35\*" -Destination "ServiceControl.Audit.Persistence.RavenDb/bin/Release/net472" -Recurse
-
-          New-Item -ItemType Directory -Force -Path ServiceControl.Monitoring/bin/Release/net472
-          Copy-Item -Path "unzip\Particular.ServiceControl.Monitoring\ServiceControl.Monitoring\*" -Destination "ServiceControl.Monitoring\bin\Release\net472" -Recurse
+          dotnet-version: 6.0.x
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.1
       - name: Build dockerfiles
-        shell: pwsh
-        run: |
-          Get-ChildItem -Path .\src\docker\ -Filter "${{ matrix.filter }}" | ForEach-Object {
-              $dockerImageName = $_.Name.SubString(0, $_.Name.Length - ".dockerfile".Length);
-              
-              $dockerbuildcmd = "docker build -t particular/" + $dockerImageName +":${{ needs.release.outputs.semver }}" + " -f .\src\docker\" + $_.Name + " ."
-              Invoke-Expression $dockerbuildcmd.ToLower()
-          }
-      - name: Login to Docker Hub
-        uses: docker/login-action@v2.1.0
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+        run: msbuild src/ServiceControl.DockerImages/ServiceControl.DockerImages.csproj -p:Configuration=Release -restore
```

Comment:
I think you can use dotnet directly and don't need to pull in msbuild

Reply:
Thanks, Daniel. I think it's correct. For now, we decided to keep it consistent with all the other places in GitHub Actions were we build ServiceControl and related stuff.